src/pyansys_sphinx_theme/__init__.py

```diff
@@ -39,3 +41,122 @@ def setup(app):
         "parallel_read_safe": True,
         "parallel_write_safe": True,
     }
+
+
+latex_elements = {
+    "preamble": r"""
+\makeatletter
+    \renewcommand{\sphinxmaketitle}{%
+        \newsavebox\imagebox
+        \newsavebox\myTitle
+        % native LaTeX command. Specifies a page with no formatting.
+        % Page number restarts after this page
+        \begin{titlepage}
+        % san serif
+        \sffamily
+        %
+        % set to 0 so that we don't have any blank space at the top of the page
+        \topskip0pt
+        %
+        % removes all margins
+        \advance\voffset -1in %
+        \advance\hoffset -1in %
+        \advance\vsize 2in %
+        \advance\hsize 2in %
+        %
+        % load the graphics into a LaTeX (not TeX) "savebox". We use this load
+        % to obtain the height later, and to insert it into the doc later
+        \sbox{\imagebox}{\includegraphics{drawing}}%
+        %
+        % put the picture in a zero sized box to get a "background effect"
+        % everything else will draw over it
+        \hbox to 0in {\vbox to 0in {\usebox{\imagebox}}}
+        %
+        % how it worked before savebox method
+        %\hbox to 0in {\vbox to 0in {\includegraphics{drawing}}} %
+        % Where the right boundary of the title block goes
+        %
+        % Define "ansysBlockWidth" <-- exactly what its name implies
+        \newdimen\ansysBlockWidth\ansysBlockWidth = 3in %
+        % Define "ansysBlockDistanceFromRightEdge" <-- exactly what its name implies
+        \newdimen\ansysBlockDistanceFromRightEdge \ansysBlockDistanceFromRightEdge = 0.5in %
+        % Define "distanceToBottomOfAnsysBox" <-- length from top of page to bottom of title box
+        \newdimen\distanceToBottomOfAnsysBox %
+        \distanceToBottomOfAnsysBox=\ht\imagebox %
+        \advance\distanceToBottomOfAnsysBox -\lineskip %
+        \advance\distanceToBottomOfAnsysBox -0.125in %
+        % Define "myHBoxWidth" <-- distance from left edge of page to right edge Ansys box
+        \newdimen\myHBoxWidth
+        \myHBoxWidth=\hsize \advance\myHBoxWidth-\ansysBlockDistanceFromRightEdge
+        % implement the Ansys block
+        \hbox to \myHBoxWidth {\vbox to \distanceToBottomOfAnsysBox {}  %
+            \hfil
+            %
+            % draw the box
+            \parbox[b]{\ansysBlockWidth}{\raggedleft {
+                \includegraphics{ansys_logo_1}} \\
+                \footnotesize
+                \textcolor{lightgray}
+                \hrule
+                \vskip 6pt
+                \textcopyright{} 2022 ANSYS, Inc. or affiliated companies\\
```

Comment:
This date could be generated at execution time. Consider using the `datetime` module for generating the value of the year.